drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_structs.h

```diff
@@ -430,6 +429,7 @@ struct wifi_nrf_fmac_priv {
  * @lmac_ver: LMAC version information.
  * @num_sta: Present number of STAs created on the device.
  * @num_ap: Present number of APs created on the device.
+ * @twt_sleep_status: current RPU sleep status.
```

Comment:
Use capital letter for first word of a sentence. In this case Current instead of current.